### PR TITLE
+ Indexes for Maps (2019)

### PR DESCRIPTION
**Pull request by diveyez**
Diyala Iraq [x] Done 9/11/19

    Avgani Index [x] Done 9/11/19

    Arc Farkhar [x] Done 9/11/19

    HazarKot Valley [x] Done 9/11/19

    Mogadishu [x] Done 9/11/19

    Esbekistan [x] Done 9/11/19

    FRL Sbenah [x] Done 9/12/19

    Tora Bora [x] Done 9/12/19

    Northen Warizistan Pakistan [x] Done 9/12/19

    Reshmaan Province [x] Done 9/12/19

    FRL Kunduz CUP [x] Rework Object/Item Config

    Fallujah [x] Done 9/12/19

    Lingor [x] Done 9/12/19 (Version 3.8+x Rework)

    G.O.S. N'Ziwasogo [x] 9/14/2019 (Fixed Civ-Mil Locations- Minor displacement in old index)

**Many More To Come**

### CODE REVIEW

**Comment by DavisBrown723** [comment]
Probably best handled by @friznit or or @marceldev89

File: addons/civ_placement/auto_civClusterGeneration.fsm [hunk 1]
```diff
@@ -8,15 +8,15 @@ item3[] = {"SWITCH",2,4346,-191.784912,-93.250328,-101.784897,-43.250317,0.00000
...
+		class ItemInfo
+		{
+			FontFace="Arial";
+			FontHeight=10;
+			lStyle=1;
+		};
+	};
 };
 link0[] = {0,1};
 link1[] = {1,3};
```
Comment (marceldev89) [inline addons/civ_placement/auto_civClusterGeneration.fsm]: These changes shouldn't be here.

File: addons/civ_placement/tests/auto_clusterGeneration.sqf [hunk 2]
```diff
@@ -1,6 +1,6 @@
 // ----------------------------------------------------------------------------
 
-#include "\x\alive\addons\civ_placement\script_component.hpp"
+#include <\x\alive\addons\civ_placement\script_component.hpp>
 SCRIPT(auto_clusterGeneration);
 
 //execVM "\x\alive\addons\civ_placement\tests\auto_clusterGeneration.sqf"
```
Comment (marceldev89) [inline addons/civ_placement/tests/auto_clusterGeneration.sqf]: These changes shouldn't be here.

File: addons/fnc_analysis/auto_gridMapAnalysis.fsm [hunk 3]
```diff
@@ -8,15 +8,15 @@ item3[] = {"SWITCH",2,4346,-191.784912,-93.250328,-101.784897,-43.250317,0.00000
...
+		class ItemInfo
+		{
+			FontFace="Arial";
+			FontHeight=10;
+			lStyle=1;
+		};
+	};
 };
 link0[] = {0,1};
 link1[] = {1,3};
```
Comment (marceldev89) [inline addons/fnc_analysis/auto_gridMapAnalysis.fsm]: These changes shouldn't be here.

File: addons/fnc_analysis/tests/auto_appendClustersMil.sqf [hunk 4]
```diff
@@ -78,119 +78,119 @@ private ["_worldName","_file","_exportString","_gridData","_cluster","_clusterCe
...
 
-    "ALiVEClient" callExtension format['sectorData~%1|%2|[ALIVE_gridData, "%3", _sectorData] call ALIVE_fnc_hashSet;',worldname,"mil",_sectorID];
+	"ALiVEClient" callExtension format['sectorData~%1|%2|[ALIVE_gridData, "%3", _sectorData] call ALIVE_fnc_hashSet;',worldname,"mil",_sectorID];
 
 } forEach (_gridData select 1);
 
 ["Adjustment complete, military results have been written to file"] call ALIVE_fnc_dump;
 
-nil;
+nil;
```
Comment (marceldev89) [inline addons/fnc_analysis/tests/auto_appendClustersMil.sqf]: These changes shouldn't be here.

File: addons/fnc_analysis/tests/auto_runMapAnalysis.sqf [hunk 5]
```diff
@@ -44,15 +44,9 @@ diag_log format["Timer End %1",_timeEnd];
...
-
-call compile preprocessFileLineNumbers _filetest;
-
-diag_log format["COMPILED STATIC DATA - Civ Construction: %1", ALIVE_civilianConstructionBuildingTypes];
-
-diag_log format["RUNNING AUTO GRID MAP ANALYSIS SQF NOW... %1", time];
+_file = format["@ALiVE\indexing\%1\x\alive\addons\main\static\%1_staticData.sqf", worldName];
+call compile preprocessFileLineNumbers _file;
 
 _FSMtest = [] execFSM "\x\alive\addons\fnc_analysis\auto_gridMapAnalysis.fsm";
```
Comment (marceldev89) [inline addons/fnc_analysis/tests/auto_runMapAnalysis.sqf]: These changes shouldn't be here.

File: addons/mil_placement/auto_milClusterGeneration.fsm [hunk 6]
```diff
@@ -8,15 +8,15 @@ item3[] = {"SWITCH",2,4346,-191.784912,-93.250328,-101.784897,-43.250317,0.00000
...
+		class ItemInfo
+		{
+			FontFace="Arial";
+			FontHeight=10;
+			lStyle=1;
+		};
+	};
 };
 link0[] = {0,1};
 link1[] = {1,3};
```
Comment (marceldev89) [inline addons/mil_placement/auto_milClusterGeneration.fsm]: These changes shouldn't be here.

File: addons/mil_placement/tests/auto_clusterGeneration.sqf [hunk 7]
```diff
@@ -1,6 +1,6 @@
 // ----------------------------------------------------------------------------
 
-#include "\x\alive\addons\mil_placement\script_component.hpp"
+#include <\x\alive\addons\mil_placement\script_component.hpp>
 SCRIPT(auto_clusterGeneration);
 
 //execVM "\x\alive\addons\mil_placement\tests\auto_clusterGeneration.sqf"
```
Comment (marceldev89) [inline addons/mil_placement/tests/auto_clusterGeneration.sqf]: These changes shouldn't be here.

File: addons/civ_placement/auto_civClusterGeneration.fsm (hunk 1 above)
Comment (diveyez) [reply addons/civ_placement/auto_civClusterGeneration.fsm]: Yes they work nicely, performance factor was great, but you are right. It looks like I will need to rework a few.

Comment (diveyez) [reply addons/civ_placement/auto_civClusterGeneration.fsm]: I may have indented the lines along the way at some point. I am sorry, I will clean this up.

File: addons/civ_placement/tests/auto_clusterGeneration.sqf (hunk 2 above)
Comment (diveyez) [reply addons/civ_placement/tests/auto_clusterGeneration.sqf]: I will correct mistakes, thank you for helping me along the way.

File: addons/civ_placement/auto_civClusterGeneration.fsm (hunk 1 above)
Comment (marceldev89) [reply addons/civ_placement/auto_civClusterGeneration.fsm]: No worries. :smiley:

**Comment by diveyez** [comment]
This pull is abandoned due to a file transfer mishap during lunch. Sorry about that guys.

**Pull request closed without merging**